asistente_ladm_col/config/db_mapping_config.py

```diff
@@ -445,6 +459,281 @@ class DBMappingConfig:
+                "LADM_COL.LADM_Nucleo.ObjetoVersionado.Fin_Vida_Util_Version": "VERSIONED_OBJECT_T_END_LIFESPAN_VERSION_F"
+            }},
+            "Modelo_Aplicacion_LADMCOL_Lev_Cat.Levantamiento_Catastral.LC_InteresadoContacto": {QueryNames.VARIABLE_NAME: "LC_PARTY_CONTACT_T", QueryNames.FIELDS_DICT: {
+                'Modelo_Aplicacion_LADMCOL_Lev_Cat.Levantamiento_Catastral.LC_InteresadoContacto.Autoriza_Notificacion_Correo': 'LC_PARTY_CONTACT_T_ALLOW_MAIL_NOTIFICATION_F',
+                'Modelo_Aplicacion_LADMCOL_Lev_Cat.Levantamiento_Catastral.LC_InteresadoContacto.Correo_Electronico': 'LC_PARTY_CONTACT_T_EMAIL_F',
+                'Modelo_Aplicacion_LADMCOL_Lev_Cat.Levantamiento_Catastral.LC_InteresadoContacto.Domicilio_Notificacion': 'LC_PARTY_CONTACT_T_NOTIFICATION_ADDRESS_F',
+                'Modelo_Aplicacion_LADMCOL_Lev_Cat.Levantamiento_Catastral.LC_InteresadoContacto.Telefono1': 'LC_PARTY_CONTACT_T_TELEPHONE_NUMBER_1_F',
+                'Modelo_Aplicacion_LADMCOL_Lev_Cat.Levantamiento_Catastral.LC_InteresadoContacto.Telefono2': 'LC_PARTY_CONTACT_T_TELEPHONE_NUMBER_2_F',
+                'Modelo_Aplicacion_LADMCOL_Lev_Cat.Levantamiento_Catastral.lc_interesado_interesadocontacto.lc_interesado..Modelo_Aplicacion_LADMCOL_Lev_Cat.Levantamiento_Catastral.LC_Interesado': 'LC_PARTY_CONTACT_T_LC_PARTY_F'
+            }},
+            "LADM_COL.LADM_Nucleo.col_puntoCcl": {QueryNames.VARIABLE_NAME: "POINT_BFS_T", QueryNames.FIELDS_DICT: {}},
+            "LADM_COL.LADM_Nucleo.COL_AreaValor": {QueryNames.VARIABLE_NAME: "COL_VALUE_AREA_T", QueryNames.FIELDS_DICT: {}},
+            "LADM_COL.LADM_Nucleo.col_masCcl": {QueryNames.VARIABLE_NAME: "MORE_BFS_T", QueryNames.FIELDS_DICT: {}},
+            "LADM_COL.LADM_Nucleo.col_menosCcl": {QueryNames.VARIABLE_NAME: "LESS_BFS_T", QueryNames.FIELDS_DICT: {}},
+            "Modelo_Aplicacion_LADMCOL_Lev_Cat.Levantamiento_Catastral.LC_TipologiaConstruccion": {QueryNames.VARIABLE_NAME: "LC_TIPOLOGY_BUILDING_T", QueryNames.FIELDS_DICT: {}},
+            "Modelo_Aplicacion_LADMCOL_Lev_Cat.Levantamiento_Catastral.LC_CalificacionNoConvencional": {QueryNames.VARIABLE_NAME: "LC_NON_CONVENTIONAL_QUALIFICATION_T", QueryNames.FIELDS_DICT: {}},
+            "Modelo_Aplicacion_LADMCOL_Lev_Cat.Levantamiento_Catastral.LC_CalificacionConvencional": {QueryNames.VARIABLE_NAME: "LC_CONVENTIONAL_QUALIFICATION_T", QueryNames.FIELDS_DICT: {}},
+            "Modelo_Aplicacion_LADMCOL_Lev_Cat.Levantamiento_Catastral.LC_GrupoCalificacion": {QueryNames.VARIABLE_NAME: "LC_GROUP_QUALIFICATION_T", QueryNames.FIELDS_DICT: {}},
+            "Modelo_Aplicacion_LADMCOL_Lev_Cat.Levantamiento_Catastral.LC_ObjetoConstruccion": {QueryNames.VARIABLE_NAME: "LC_BUILDING_OBJECT_T", QueryNames.FIELDS_DICT: {}},
+            "Modelo_Aplicacion_LADMCOL_Lev_Cat.Levantamiento_Catastral.LC_DatosAdicionalesLevantamientoCatastral": {QueryNames.VARIABLE_NAME: "LC_SURVEY_CADASTRAL_ADITIONAL_DATA_T", QueryNames.FIELDS_DICT: {}},
```

Comment:
LC_SURVEY_CADASTRAL_ADITIONAL_DATA_T replace by LC_SURVEY_CADASTRAL_ADDITIONAL_DATA_T

Reply:
fixed in https://github.com/SwissTierrasColombia/Asistente-LADM-COL/pull/577/commits/4888b2f347196a8dc65bad715dc7f8c71a5731dd